service/actions/helpers.js

```diff
@@ -12,7 +12,7 @@ export async function forceLogoutIfNecessary(err, dispatch) {
     }
 }
 
-function dispatcher(type, data, dispatch, getState) {
+export function dispatcher(type, data, dispatch, getState) {
```

Comment:
I'm exporting it because I'm calling it manually in the preferences actions. The preferences actions just return "true" from their API so I needed to reimplement bindClientFunc to get around that